visual_behavior/data_access/loading.py

```diff
@@ -133,6 +133,28 @@ def get_visual_behavior_cache(manifest_path=None):
     return cache
 
 
+def get_released_ophys_experiment_table(exclude_ai94=True):
+    '''
+    gets the released ophys experiment table from AWS
+
+    Keyword Arguments:
+        exclude_ai94 {bool} -- If True, exclude data from mice with Ai94(GCaMP6s) as the reporter line. (default: {True})
+
+    Returns:
+        experiment_table -- returns a dataframe with ophys_experiment_id as the index and metadata as columns.
+    '''
+    data_storage_directory = '/allen/programs/braintv/workgroups/nc-ophys/visual_behavior/production_cache'
```

Comment:
Could this line be written to accommodate windows users? I have been using 
`if os.name == 'nt':
    fig_path = "//allen/programs/braintv/workgroups/nc-ophys/visual_behavior"
else:
    fig_path = "/allen/programs/braintv/workgroups/nc-ophys/visual_behavior"`. Maybe there is a better solution?

Reply:
Thanks for pointing that out! Simply making the first '/' into a '//' will make it platform independent. I just committed a change to address this.